src/server_manager/web_app/app.ts

```diff
@@ -757,6 +772,7 @@ export class App {
     view.serverHostname = selectedServer.getHostname();
     view.serverManagementApiUrl = selectedServer.getManagementApiUrl();
     view.serverPortForNewAccessKeys = selectedServer.getPortForNewAccessKeys();
+    view.isAccessKeyPortEditable = await isAccessKeyPortEditable(selectedServer);
```

Comment:
Won't this block the UI? Can we do this where we get the server information instead and make the showServer a sync method again?

Reply:
Done.  We now use the version field in `shadowbox/package.json` to report shadowbox version from the management API.